Makefile.vc

```diff
@@ -0,0 +1,266 @@
+#
+# Options:
+# machine=Win32|Win64
+# prefix=\path\to\install
+#
+# If no machine is specified, the makefile uses the systems
+# PROCESSOR_ARCHITECTURE.
+#
+# Compiling:  nmake -f Makefile.vc
+# Installing: nmake -f Makefile.vc prefix=\path\to\x install
+#
+
+VERSION=0.1.5
+
+# Compiler, linker, librarian, and other tools
+CC  = cl
+CXX = cl
+LD  = link
+AR  = lib
+
+!IFDEF MACHINE
+!IF "$(MACHINE)" == "Win64"
+TARGET_FLAGS = /D__x86_64__
```

Comment:
I think this whole block of setting defines is unneeded here, or am I missing something? In that case, some lines in the comment at the top of the file can be removed as well.